setup.py

```diff
@@ -43,7 +43,7 @@ def run_tests(self):
         'python-dateutil',
     ],
     extras_require={
-        'presto': ['requests>=1.0.0'],
+        'presto': ['requests>=1.0.0', 'requests_kerberos>=0.12.0'],
```

Comment:
This is actually only needed when using Kerberos, wouldn't it be better to add an extra `extra`?

Reply:
I have added a `kerberos` field in `extra_require`. I think that is more suitable. Please let me know your thoughts on this.